projects/cashmere/src/lib/sass/scroll-nav.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../sass/colors';
+@import '../../sass/mixins';
```

Comment:
I think this is what's causing the build error.  I think it should be:
`@import './colors';
@import './mixins';`

Reply:
went ahead and pushed an update to this branch to resolve the error